### PR TITLE
(WIP) benchmarks.md

### PR DESCRIPTION
**Pull request by Titaniumtown**
I figured all the commit stuff out. lol 
successor to #55 and #146

### CODE REVIEW

**Comment by Titaniumtown** [comment]
I'm doing the benchmarks as we speak. Don't merge this yet!!!

**Comment by Chew** [comment]
Draft pr draft pr draft pr

**Comment by Titaniumtown** [comment]
oh yea, should of done that. Can I convert a pre-existing PR to a draft?

**Comment by Chew** [comment]
image

**Comment by Titaniumtown** [comment]
I have my web browser like half size, I didn't see that. lol

**Comment by Titaniumtown** [comment]
I think i screwed up testing yatopia 1.16.2, imma retest that rn.

**Comment by Titaniumtown** [comment]
I did ```chm generate world 2000``` instead of ```chm generate world 1000``` lol

**Comment by duplexsystem** [comment]
In theory that shouldn't matter but in practice it probably does matter because minecraft is BS.

**Comment by MrIvanPlays** [comment]
Make sure to redo the tests with https://github.com/YatopiaMC/Yatopia/commit/e1a085de4c7559a982a915d1b199d9a3a079f38b as we have the latest improvements.

**Comment by Titaniumtown** [comment]
will do @MrIvanPlays

**Comment by Titaniumtown** [comment]
I'm benchmarking e1a085de4c7559a982a915d1b199d9a3a079f38b rn

**Comment by Titaniumtown** [comment]
I will continue testing later today after school (3:30pm EST) and when I have some breaks in between classes, as I am doing E-Learning

**Comment by toinouH** [comment]
Yatopia 1.16.2 should be the latest that you should benchmark, it would be easier for you to be up-to-date.

**Comment by duplexsystem** [comment]
anymore news on this?

**Comment by Titaniumtown** [comment]
been busy w/ school, (TONS of homework) I can continue/update the benchmarks this weekend.

**Comment by toinouH** [comment]
He is very busy for the moment.

**Comment by duplexsystem** [comment]
I'm just checking

**Comment by Titaniumtown** [comment]
It's good :)

File: benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Chunkgen benchmarks
```
Comment (MrIvanPlays) [inline benchmarks.md:1]: **WARNING: THE FOLLOWING BENCHMARKS SHOULD NOT BE USED FOR ANY DEBATE, THEY'RE ONLY HERE FOR REFERENCE PURPOSES. DIFFERENT HARDWARE MIGHT AND WILL GIVE DIFFERENT RESULTS.**

**Comment by Titaniumtown** [comment]
I am going to close this PR because at this time I no longer have the time to update all the benchmarks and keep them up-to-date. I hope someone else has the time to do benchmarks in the future! :)

**Pull request closed without merging**